packages/FlexGrid/FlexGrid.jsx

```diff
@@ -17,16 +16,8 @@ import styles from './FlexGrid.modules.scss'
  *
  * @version ./package.json
  */
-const FlexGrid = ({ centre, limitWidth, gutter, children, ...rest }) => {
-  if (centre) {
-    deprecate(
-      'core-flex-grid',
-      'The centre prop is deprecated due to the limitWidth prop centring the grid on its own. Please remove the centre prop from your grid definition.'
-    )
-  }
-
-  const getClasses = () =>
-    joinClassNames(styles.flexGrid, centre && styles.centre, limitWidth && styles.limitWidth)
+const FlexGrid = ({ limitWidth, gutter, children, ...rest }) => {
+  const getClasses = () => joinClassNames(styles.flexGrid, limitWidth && styles.limitWidth)
```

Comment:
Small thing. The `getClasses` function is not needed.... Just call `joinClassNames` directly. No need for the extra indirection.